lib/internal/util/inspect.js

```diff
@@ -535,6 +535,14 @@ function getEmptyFormatArray() {
   return [];
 }
 
+function isInstanceof(object, proto) {
+  try {
+    return object instanceof proto;
+  } catch {
+    return false;
+  }
+}
+
```

Comment:
why is the try/catch needed? are we worried about someone defining a `Symbol.hasInstance` method that can throw?

Reply:
Based on https://github.com/nodejs/node/pull/36178/commits/cb9eec9f079949c99ac5f32bcd36b90dec07bd11#r527685177, `instanceof` can throw

Reply:
ah, didn't realize this was a backport PR. turns out `instanceof` can indeed throw when the RHS is not a function, or when it's a non-constructible function (like an arrow function or a concise method)